design/resources.go

```diff
@@ -110,8 +110,8 @@ var _ = a.Resource("workitem.2", func() {
 		a.Description("List work items.")
 		a.Params(func() {
 			a.Param("filter", d.String, "a query language expression restricting the set of found work items")
-			a.Param("page[offset]", d.Number, "Paging in the format <start>,<limit>")
-			a.Param("page[limit]", d.Number, "Paging in the format <start>,<limit>")
+			a.Param("page[offset]", d.String, "Paging start position")
```

Comment:
See comment on the issue: "The reason for offset being a string is that while we currently use the row offset(int) of the result set, it's not unlikely to have an offset of some natural order key, e.g. date. Give me 100(limit) starting from 2010-01-02(offset)."